programs/compressed-token/src/process_mint.rs

```diff
@@ -360,12 +340,13 @@ pub struct MintToInstruction<'info> {
     pub fee_payer: Signer<'info>,
     #[account(mut)]
     pub authority: Signer<'info>,
-    // This is the cpi signer
+    // This is the cpi signer.
+    // TODO: double check security
     /// CHECK: that mint authority is derived from signer
-    #[account(mut, seeds = [MINT_AUTHORITY_SEED, authority.key().to_bytes().as_slice(), mint.key().to_bytes().as_slice()], bump,)]
-    pub mint_authority_pda: UncheckedAccount<'info>,
+    #[account(seeds = [b"cpi_authority"], bump,)]
```

Comment:
don't we have a constant for this?

Reply:
will do in follow-up pr with all other occurrences